frontend/src/entry/settings/ManualEntityInput.tsx

```diff
@@ -16,7 +16,8 @@ export default function ManualEntityInput({ onAddEntityManually, errorMessage }:
 
   const { uiConfig, isUiConfigLoading, isUiConfigError } = useGetUiConfig()
 
-  const handleAddEntityManuallyOnClick = async () => {
+  const handleAddEntityManuallyOnClick = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
```

Comment:
It's not always clear what behaviour we're trying to prevent when doing this sort of thing. Can we add a brief comment to explain?

Reply:
This stops the page from refreshing when you submit the form